src/reducers/vis-state-merger.js

```diff
@@ -213,13 +213,15 @@ export function mergeInteractions(state, interactionToBeMerged) {
       merged[key] = {
         ...state.interactionConfig[key],
         enabled,
-        config: pick(
-          {
-            ...state.interactionConfig[key].config,
-            ...configToMerge
-          },
-          Object.keys(state.interactionConfig[key].config)
-        )
+        ...(state.interactionConfig[key].config ? {
```

Comment:
coordinate entry doesn't have a config property

Reply:
can you add tests in vis-schema to test for
1. coordinates are saved and correctly loaded
2. older config without coordinates will result in loading error

Reply:
@heshan0131 are you sure old configuration should fail? Would it be better if we handle them gracefully and default to __coordinate enabled: false__?

Reply:
let's make sure old configs are not triggering loading errors